package.json

```diff
@@ -30,7 +30,7 @@
   },
   "author": "Matteo Collina <hello@matteocollina.com>",
   "engines": {
-    "node": ">=10"
+    "node": ">=16"
```

Comment:
Two reasons for this change: 
* CI already was not executed on 10 and 12 before this change
* Newer TypeScript linter doesn't work on Node < 16. If for whatever reason it is important to preserve compatibility with Node 14, I can separate CI runs for linting and tests, but 14 has been EOL for a while anyway.